ludwig/schema/features/utils.py

```diff
@@ -86,6 +87,13 @@ def get_input_feature_conds():
         feature_schema = schema_utils.unload_jsonschema_from_marshmallow_class(schema_cls)
         feature_props = feature_schema["properties"]
         schema_utils.remove_duplicate_fields(feature_props)
+
+        # TODO: Is this the best way?  Discuss with ksbrar
+        # special handling for image augmentation
+        if feature_type == IMAGE:
```

Comment:
I don't think this is necessary, because by including `AugmentationContainerDataclassField` directly on the image feature schema it should already have the augmentation props.